flink-end-to-end-tests/test-scripts/common.sh

```diff
@@ -305,6 +305,10 @@ function wait_dispatcher_running {
 }
 
 function start_cluster {
+  # After FLINK-24785, RocksDB's log would be created under Flink's log directory by default,
+  # this would make e2e tests' artifacts containing too many log files.
+  # As RocksDB's log would not help much in e2e tests, move the location to the '/tmp' folder.
+  set_config_key "state.backend.rocksdb.log.dir" "/tmp"
```

Comment:
is there no way to disable the logging altogether? 🤔

Reply:
isn't that like, really bad? Surely users would want some easy way to opt-out of rocks db dumping mountains of files into the log directory :/

Reply:
If you want the logs still created under the DB folder, we can set this options to a non-existing folder, such as `/dev/null`. This will make the behavior same as before. Is this what you want?

Reply:
If relocating to RocksDB's own folder, the folder itself will be cleaned up when the RocksDB state-backend disposed just as before. In other words, we don't need to manually clean up the log files but the parent folder will be cleaned up automatically.